Static Analysis fails

### What happened?

The PR https://github.com/etcd-io/etcd/pull/15281 is causing a format error. Just execute "./scripts/fix.sh" should can fix it. cc @jmhbnz @chaochn47 

```
gofmt' started at Mon Apr 10 22:34:42 UTC 2023
fileutil/preallocate.go
FAIL: 'go_fmt_for_package ./...' checking failed (printed output)
diff fileutil/preallocate.go.orig fileutil/preallocate.go
FAIL: 'gofmt' failed at Mon Apr 10 22:34:42 UTC 2023
--- fileutil/preallocate.go.orig
+++ fileutil/preallocate.go
@@ -19,8 +19,8 @@
make: *** [Makefile:73: verify-gofmt] Error 255
 	"os"
 )
 
-// Preallocate tries to allocate the space for given file. This 
-// operation is only supported on darwin and linux by a few 
+// Preallocate tries to allocate the space for given file. This
+// operation is only supported on darwin and linux by a few
 // filesystems (APFS, btrfs, ext4, etc.).
 // If the operation is unsupported, no error will be returned.
 // Otherwise, the error encountered will be returned.
Error: Process completed with exit code 2.

```

https://github.com/etcd-io/etcd/actions/runs/4661867210/jobs/8251618403



### What did you expect to happen?

see above

### How can we reproduce it (as minimally and precisely as possible)?

See https://github.com/etcd-io/etcd/actions/runs/4661867210/jobs/8251618403

### Anything else we need to know?

_No response_

### Etcd version (please run commands below)

<details>

```console
$ etcd --version
# paste output here

$ etcdctl version
# paste output here
```

</details>


### Etcd configuration (command line flags or environment variables)

<details>

# paste your configuration here

</details>


### Etcd debug information (please run commands below, feel free to obfuscate the IP address or FQDN in the output)

<details>

```console
$ etcdctl member list -w table
# paste output here

$ etcdctl --endpoints=<member list> endpoint status -w table
# paste output here
```

</details>


### Relevant log output

_No response_

--- a/client/pkg/fileutil/preallocate.go
+++ b/client/pkg/fileutil/preallocate.go
@@ -19,8 +19,8 @@ import (
 	"os"
 )
 
-// Preallocate tries to allocate the space for given file. This 
-// operation is only supported on darwin and linux by a few 
+// Preallocate tries to allocate the space for given file. This
+// operation is only supported on darwin and linux by a few
 // filesystems (APFS, btrfs, ext4, etc.).
 // If the operation is unsupported, no error will be returned.
 // Otherwise, the error encountered will be returned.

